packages/teleport/src/Player/DesktopPlayer.tsx

```diff
@@ -141,20 +94,40 @@ const useDesktopPlayer = ({
     canvas: HTMLCanvasElement,
     spec: ClientScreenSpec
   ) => {
+    const styledPlayer = canvas.parentElement;
```

Comment:
Do we need to run this logic on every PNG frame? Or can we instead run it once when we initialize the player and (optionally) run it in a window resize handler?

Reply:
It's only being run when receive the `client screen spec` tdp message, so since we currently don't support a dynamic resize, it just happens once at the beginning. Once we do implement resize, this will also account for that.

Reply:
Oh! Github's UI confuses me, I looked up and it looked like this was inside `tdpCliOnPngFrame`, but it's not. My eyes always glance over the light blue bars. Sorry!